content/base-image-packages.md

```diff
@@ -22,6 +35,31 @@ The Galaxy base image runs Ubuntu 14.04 and comes with a set of packages pre-ins
 - [xfonts-base](http://packages.ubuntu.com/trusty/xfonts-base)
 - [xfonts-75dpi](http://packages.ubuntu.com/trusty/xfonts-75dpi)
 - [libpoppler-qt4-dev](http://packages.ubuntu.com/trusty/libpoppler-qt4-dev)
-- [git](http://packages.ubuntu.com/trusty/git)
 
-When Galaxy builds your deployed app into a Docker container, it installs the same version of Node used by the version of the `meteor` command-line tool that you used to deploy it.  (Currently, this uses the official binary distribution of Node from nodejs.org, not the Ubuntu package. It is installed into a non-standard directory which is prepended to the `$PATH` that your app sees.)
+In addition, the default base image contains several popular versions of Node preinstalled in directories with names like `/node-v8.9.3-linux-x64`. (Including these versions makes the container build and start process more efficient.)  Galaxy uses the official binary distribution of Node from nodejs.org, not the Ubuntu package.
+
+<h2 id="build">Build time behavior</h2>
+
+When you run `meteor deploy`, your local `meteor` command-line tool builds your app, bundles it into a [tarball](https://en.wikipedia.org/wiki/Tar_(computing%29), and uploads it to Galaxy.  Galaxy then builds a Docker image for your app based on the default base image by running the `/app/setup.sh` script inside the image.  This section describes exactly what the default base image's `/app/setup.sh` script does.
```

Comment:
This line appears to be rendering incorrectly - link not showing. You want a link where link is but it's literally showing [link](http://link.com) as the user actually views it. There may be a space between [tarball] and (link) that's causing this.

Reply:
I'm not sure what you're referring to. Can you show me where you're seeing this? When I run the site with `npm run start` it looks correct. Show me on your screen or mine?